ReactCommon/cxxreact/Value.h

```diff
@@ -13,6 +13,7 @@
 #include <JavaScriptCore/JSValueRef.h>
 
 #include <folly/dynamic.h>
+#include <jni/LocalString.h>
```

Comment:
I think we're fine with just copying this code into cxxreact/ for now. This fixes one of the most common crashes we've seen in the wild. Longer term, we'll figure out a less bad solution... but that may be a while.